projects/fsastorefront-e2e-cypress/cypress/helpers/checkout/group-policy.ts

```diff
@@ -66,17 +66,17 @@ export function createNewMember() {
   register.createB2bCustomer(createCustomer);
 }
 
-export function changePasswordForNewMember() {
-  cy.contains(createCustomer.firstName).click();
+export function checkMemberIsDisabled() {
+  cy.contains(createCustomer.firstName)
+    .parentsUntil('table')
+    .within(() => {
+      cy.get('cx-org-status-cell').should('contains.text', 'Disabled').click();
```

Comment:
Is there a use for this click here since button is disabled?

Reply:
Yes, i am using this disabled as to click on the row, so the details about the new member will be displayed